src/course/en/course.json

```diff
@@ -89,14 +89,17 @@
             "accessibilityToggleTextOn": "Turn accessibility on?",
             "accessibilityToggleTextOff": "Turn accessibility off?",
             "_accessibilityInstructions": {
-                "touch": "Usage instructions. Use swipe right for next. Use swipe left for previous. Use a double tap to select. Use a two finger slide up to go to the top of the page.",
+                "touch": "Usage instructions. Use swipe left for next. Use swipe right for previous. Use a double tap to select. Use a two finger slide up to go to the top of the page.",
                 "notouch": "Usage instructions. Use tab for next. Use shift tab for previous. Use enter to select. Use escape to go to the top of the page.",
-                "ipad": "Usage instructions for touchscreens. Use swipe right for next. Use swipe left for previous. Use a double tap to select. Use a two finger slide up to go to the top of the page. Usage instructions for keyboard access. Use right for next. Use left for previous. Use up and down together to select."
+                "ipad": "Usage instructions for touchscreens. Use swipe left for next. Use swipe right for previous. Use a double tap to select. Use a two finger slide up to go to the top of the page. Usage instructions for keyboard access. Use left for next. Use right for previous. Use up and down together to select."
             },
             "_ariaLabels": {
                 "navigation": "Course navigation bar",
                 "menuLoaded": "Menu Loaded.",
                 "menu": "Menu",
+                "menuItem": "Menu item.",
+                "menuViewButton": "Select here to enter.",
+                "menuEnd": "You have reached the end of the menu.",
```

Comment:
Can confirm it's definitely the `_boxmenu` versions of `menuItem` and `menuEnd` that are used in the output - and that `_ariaLabels.menuViewButton` isn't being used.